components/support/nimbus-cli/src/output/info.rs

```diff
@@ -0,0 +1,230 @@
+            if app_name != params.app_name {
+                continue;
+            }
+
+            let info = match ExperimentInfo::try_from(&exp) {
+                Ok(e) => e,
+                _ => continue,
+            };
+
+            let is_rollout = if info.is_rollout { "R" } else { "" };
+
+            term.write_line(&format!(
+                " {slug: <65}| {channel: <8}| {bucketing: >5} | {features: <30}| {is_rollout: <1} | {branches}",
+                slug = info.slug,
+                channel = info.channel,
+                bucketing = info.bucketing_percent(),
+                features = info.features.join(", "),
+                branches = info.branches.join(", ")
+            ))?;
+        }
+        Ok(true)
+    }
+}
+
+impl ExperimentSource {
+    pub(crate) fn print_info<P>(&self, output: Option<P>) -> Result<bool>
+    where
+        P: AsRef<Path>,
+    {
+        let value = self.try_into()?;
+        let info: ExperimentInfo = ExperimentInfo::try_from(&value)?;
+        if output.is_some() {
+            value_utils::write_to_file_or_print(output, &info)?;
+            return Ok(true);
+        }
+        let url = match self {
+            Self::FromApiV6 { slug, endpoint } => Some(format!("{endpoint}/nimbus/{slug}/summary")),
+            _ => None,
+        };
+        let term = Term::stdout();
+        let t_style = term.style().italic();
+        let d_style = term.style().bold().cyan();
+        let line = |title: &str, detail: &str| {
+            _ = term.write_line(&format!(
+                "{: <11} {}",
+                t_style.apply_to(title),
+                d_style.apply_to(detail)
+            ));
+        };
+
+        let enrollment = format!(
+            "{} ({})",
+            info.enrollment,
+            if info.is_enrollment_paused {
+                "paused"
+            } else {
+                "enrolling"
+            }
+        );
+
+        let is_rollout = if info.is_rollout {
+            "Rollout".to_string()
+        } else {
+            let n = info.branches.len();
+            let b = if n == 1 {
+                "1 branch".to_string()
+            } else {
+                format!("{n} branches")
+            };
+            format!("Experiment with {b}")
+        };
+
+        line("Slug", info.slug);
+        line("Name", info.user_facing_name);
+        line("Description", info.user_facing_description);
+        if let Some(url) = url {
+            line("URL", &url);
+        }
+        line("App", info.app_name);
+        line("Channel", info.channel);
+        line("E/R", &is_rollout);
+        line("Enrollment", &enrollment);
+        line("Observing", &info.duration.to_string());
+        line("Targeting", &format!("\"{}\"", info.targeting));
+        line("Bucketing", &info.bucketing_percent());
+        line("Branches", &info.branches.join(", "));
+        line("Features", &info.features.join(", "));
+
+        Ok(true)
+    }
+}
```

Comment:
I may be missing them somewhere else but are there any tests for this file? I don't know if it's viable to test the terminal output but I do think it would at least be worth testing the DateRange string formatting against some examples.